arch/arm/core/aarch32/cortex_a_r/exc.S

```diff
@@ -196,58 +154,9 @@ SECTION_SUBSEC_FUNC(TEXT, __exc, z_arm_data_abort)
 	 */
 	sub lr, #8
 
-	/*
-	 * Store r0-r3, r12, lr, lr_abt and spsr_abt into the stack to
-	 * construct an exception stack frame.
-	 */
-	srsdb sp!, #MODE_ABT
-	stmfd sp, {r0-r3, r12, lr}^
-	sub sp, #24
-
-	/*
-	 * Create new esf struct for exception handler debug.  The first
-	 * time the basic stack frame is saved is for getting in and out
-	 * of the exception.
-	 */
-#if defined(CONFIG_EXTRA_EXCEPTION_INFO)
-	sub sp, #___callee_saved_t_SIZEOF
-	sub sp, #___extra_esf_info_t_SIZEOF
-#endif
-
-	srsdb sp!, #MODE_ABT
-	stmfd sp, {r0-r3, r12, lr}^
-	sub sp, #24
-
-	/* Increment exception nesting count */
-	ldr r2, =_kernel
-	ldr r0, [r2, #_kernel_offset_to_nested]
-	add r0, r0, #1
-	str r0, [r2, #_kernel_offset_to_nested]
-
-#if defined(CONFIG_EXTRA_EXCEPTION_INFO)
-	/* Pointer to extra esf info */
-	add r0, sp, #___basic_sf_t_SIZEOF
-	mov r1, #0
-	str r1, [r0, #4]
-	str r1, [r0, #8]
-
-	/* Pointer to callee saved registers */
-	add r1, r0, #___extra_esf_info_t_SIZEOF
-	str r1, [r0]
-
-	cps #MODE_SYS
-	stm r1, {r4-r11, sp}
-	cps #MODE_ABT
-#endif
-
-	/* Invoke fault handler */
-	mov r0, sp
+	exception_entry MODE_ABT
 	bl z_arm_fault_data
 
```

Comment:
Spurious empty line after `bl` (it would be nice to keep it consistent with the rest).

Reply:
fixed